spec/models/service_template_provision_task_spec.rb

```diff
@@ -86,7 +88,7 @@ def service_resource_id(index, scaling_max)
           :class_name       => 'ServiceProvision_Template',
           :instance_name    => 'clone_to_service',
           :automate_message => 'create',
-          :attrs            => {'request' => 'clone_to_service'},
+          :attrs            => {'request' => 'clone_to_service', 'Service::Service' => @service.id},
```

Comment:
@gmcculloug Discussed this with @mkanoor.  No need to down case the variable name since attributes are down cased in the engine:
https://github.com/tinaafitz/manageiq/blob/master/lib/miq_automation_engine/engine/miq_ae_object.rb#L277